src/serialize.cpp

```diff
@@ -12,10 +12,12 @@
 
```

Comment:
Regarding this error:
```
src/serialize.cpp:8:10 [clang-diagnostic-error]
'Message_generated.h' file not found
```
We would probably need to link `flatbuffers_interface` as public to the target to fix it.
We can also choose to ignore this and keep the private link if we hold on to it.